.github/workflows/build-and-deploy-to-mainnet.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build, push and deploy staking dashboard to mainnet
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_and_push:
+    name: Build and push
+    runs-on: ubuntu-20.04
+    steps:
+      - name: GIT | Checkout
+        uses: actions/checkout@master
+
+      - name: DOCKER | Build and push mainnet image
+        uses: ./.github/actions/build-and-push
+        id: build_mainnet
+        with:
+          environment: mainnet
+          build_envs: "NODE_ENV=production REACT_APP_DISABLE_FIAT=1"
```

Comment:
I think we can remove `NODE_ENV`, it's by default set to `production` if we use `npm build`
https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
Thanks for info, according to this document I have removed `NODE_ENV` from `build_envs`.